Platform/TigerlakeBoardPkg/BoardConfig.py

```diff
@@ -64,6 +64,26 @@ def __init__(self, *args, **kwargs):
         # 0: Disable  1: Enable  2: Auto (disable for UEFI payload, enable for others)
         self.ENABLE_SMM_REBASE     = 2
 
+        # TCC features
+        # To enable default TCC features, set 1 to self.ENABLE_TCC
+        self.ENABLE_TCC = 0
+        self.INCLUDE_FW_TCC = 0

```

Comment:
The fine tuning is really not necessary. Cfg settings and Tcc subregions go hand-in-hand. If either one is absent, Tcc feature doesnt work. ENABLE_TCC should be able to control both Cfg settings as well as container sub-region reservation. No need to split.

Reply:
Thanks. the INCLUDE_FW_TCC is reverted